Web/Edubase.Web.UI/Areas/Groups/Views/Group/CreateAcademyTrust.cshtml

```diff
@@ -41,7 +41,7 @@
             @if (Model.TrustExists == true)
             {
                 <div role="alert" class="warning-message js-trigger-aria-live">
-                    <p class="message-text" aria-live="polite">Company is already a @(Model.TypeId == (int)eLookupGroupType.MultiacademyTrust? "multi-academy" : "single-academy") trust</p>
```

Comment:
Would it be worth specifying the specifics of the error message for the page / content?

Reply:
yeah, sounds like a good idea to me :-)

Reply:
Change made in latest commit.